apollo-router/src/plugins/cache/entity.rs

```diff
@@ -1765,3 +1781,158 @@ impl Ord for CacheKeyStatus {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::plugins::cache::tests::{MockStore, SCHEMA};
+
+    use super::*;
+
+    #[tokio::test]
+    async fn test_subgraph_enabled() {
+        let valid_schema = Arc::new(Schema::parse_and_validate(SCHEMA, "test.graphql").unwrap());
+        let redis_cache = RedisCacheStorage::from_mocks(Arc::new(MockStore::new()))
+            .await
+            .unwrap();
+        let map = [
+            (
+                "user".to_string(),
+                Subgraph {
+                    redis: None,
+                    private_id: Some("sub".to_string()),
+                    enabled: None,
+                    ttl: None,
+                    ..Default::default()
+                },
```

Comment:
“Real” users never construct struct literals like this. Since these two test is all about configuration, I think they should instead parse configuration from a YAML string or a `json!` macro in order to exercise things like `#[serde(default)]`

Reply:
Done, thanks for the review